### ISSUE TITLE
future - tags

### ISSUE THREAD

**Issue by kanishka-work** [OCamlPro/drom#45]
Tags don't seem to be heavily used on opam. Would be nice to facilitate more use. I am not sure how to do this exactly, but this project might hekpt

**Comment by lefessan** [comment]
For now, `drom` automatically add a tag `org:` with the Github organization. 

Are you aware of any other useful tag that we could add/use ?

**Comment by kanishka-work** [comment]
Nothing automatic. I imagine it more to raise user awareness to add tags when initially creating a project. I like how you can browse hackage by tag, and if people get into habit of using opam tags, the same browsing would be possible in opam web.

**Comment by kanishka-work** [comment]
I'm thinking tags are not as useful. opam can automatically generate tags later on or just continue to rely on free text search. I think manual curation in ocamlverse/awesome-ocaml is more useful for now.

**Comment by zapashcanon** [comment]
Re-opening as I think it's a good idea. I always forgot to add tags. We could add a command which, amongs other things, suggest the user to add at least something like 3 tags.

**Comment by kanishka-work** [comment]
Makes sense, but try out opam's search. You can see its doing a pretty good job of free text indexing currently.
